src/sidebar.jsx

```diff
@@ -154,10 +156,10 @@ export const SidebarPanelDetails = ({
                                 {cockpit.format(
                                     cockpit.ngettext(
                                         "$0 item $1", "$0 items $1",
```

Comment:
The way these strings get pasted together is pretty confusing.  Of course there's no two-dimensional ngettext(), so we're going to have to do some kind of pasting, but I think I'd prefer if it looked something like:

```js
  let str = ngettext("$0 item", "$0 items", visible_count);
  if (!showHidden)
    str += " " + ngettext("($0 hidden)", "($0 hidden)", hidden_count)
```

Reply:
In fact, we could turn it around and use the translation of the "hidden" part to do the pasting — since it's the one that's only done conditionally.

If you do that, then please write a translator comment about it.